src/item.rs

```diff
@@ -210,3 +215,156 @@ fn drop_system(
         );
     }
 }
+
+/// A component that with Breakable, explodes.
+#[derive(Component, Clone)]
+pub struct Explodable {
+    pub attack: AttackMeta,
+    pub timer: Timer,
+    pub fusing: bool,
+    pub animated_sprite: AnimatedSpriteSheetBundle,
+    pub explosion_frames: AttackFrames,
+}
```

Comment:
Does explodable need to have its own `AttackMeta` and `AttackFrames` ?
`AttackMeta` already has its own `AttackFrames` but couldnt `Explodable` be a component with only `timer` and `fusing` and `animated_sprite`?
Then add the `Explodable` component to an entity with an `AttackMeta` and query for both when you need to deal with both.

Reply:
Not sure about the other thing you said, but we need two attack frames actually, one for the bomb explosion, and one for the frames the boss throws the bombs